modules/service/src/main/scala/lucuma/odb/util/Codecs.scala

```diff
@@ -65,123 +55,137 @@ trait Codecs {
     )
   }
 
-  def enumerated[A](tpe: Type)(implicit ev: Enumerated[A]): Codec[A] =
-    `enum`(ev.tag, ev.fromTag, tpe)
+  // -----
 
-  val user_id: Codec[User.Id] = gid[User.Id]
-  val target_id: Codec[Target.Id] = gid[Target.Id]
-  val program_id: Codec[Program.Id] = gid[Program.Id]
-  val observation_id: Codec[Observation.Id] = gid[Observation.Id]
+  val air_mass_range_value: Codec[PosBigDecimal] =
+    numeric(3, 2).eimap(PosBigDecimal.from)(_.value)
 
-  val site: Codec[Site] =
-    `enum`(_.tag.toLowerCase, s => Site.fromTag(s.toUpperCase), Type("e_site"))
+  val angle_µas: Codec[Angle] =
+    int8.imap(Angle.microarcseconds.reverseGet)(Angle.microarcseconds.get)
 
-  val user_type: Codec[UserType] =
-    enumerated(Type("e_user_type"))
+  val catalog_name: Codec[CatalogName] =
+    enumerated(Type("e_catalog_name"))
 
-  val program_user_role: Codec[ProgramUserRole] =
-    enumerated(Type("e_program_user_role"))
+  val cloud_extinction: Codec[CloudExtinction] =
+    enumerated[CloudExtinction](Type.varchar)
 
-  val program_user_support_type: Codec[ProgramUserSupportType] =
-    enumerated(Type("e_program_user_support_type"))
+  val data_timestamp: Codec[Timestamp] =
+    timestamp.eimap(
+      ldt => Timestamp.FromLocalDateTime.getOption(ldt).toRight(s"Invalid Timestamp: $ldt"))(
+      _.toLocalDateTime
+    )
 
-  val _site: Codec[Arr[Site]] =
-    Codec.array(_.tag.toLowerCase, s => Site.fromTag(s.toUpperCase).toRight(s"Invalid tag: $s"), Type("_e_site"))
+  val declination: Codec[Declination] =
+    angle_µas.eimap(
+      a => Declination.fromAngle.getOption(a).toRight(s"Invalid declination: $a"))(
+      Declination.fromAngle.reverseGet
+    )
+
+  val edit_type: Codec[EditType] =
+    enumerated(Type("e_edit_type"))
+
+  val ephemeris_key_type: Codec[EphemerisKeyType] =
+    enumerated(Type("e_ephemeris_key_type"))
+
+  val epoch: Codec[Epoch] =
+    varchar.eimap(
+      s => Epoch.fromString.getOption(s).toRight(s"Invalid epoch: $s"))(
+      Epoch.fromString.reverseGet
+    )
 
   val existence: Codec[Existence] =
     enumerated(Type("e_existence"))
 
-  val tag: Codec[Tag] =
-    varchar.imap(Tag(_))(_.value)
+  val hour_angle_range_value: Codec[BigDecimal] =
+    numeric(3, 2)
 
-  val obs_status: Codec[ObsStatus] =
-    enumerated(Type("e_obs_status"))
+  val image_quality: Codec[ImageQuality] =
+    enumerated[ImageQuality](Type.varchar)
+
+  val int_percent: Codec[IntPercent] =
+    int2.eimap(n => IntPercent.from(n))(_.value.toShort)
 
   val obs_active_status: Codec[ObsActiveStatus] =
     enumerated(Type("e_obs_active_status"))
 
-  // TEMPORARY; the tag will be correct soon
-  val too_activation: Codec[ToOActivation] =
-    `enum`(
-      e => e.label.toLowerCase(),
-      s => Enumerated[ToOActivation].all.find(_.label.toLowerCase == s),
-      Type("e_too_activation")
-    )
```

Comment:
This should work now since the tag is now correct in the `lucuma-core` we're using.